rules/phpmd.xml

```diff
@@ -44,7 +45,7 @@
     </rule>
     <rule ref="rulesets/naming.xml/ShortVariable">
         <properties>
-            <property name="exceptions" value="id" />
+            <property name="exceptions" value="id,io" />
```

Comment:
io als uitzondering is nooit besproken met andere leads

Reply:
Heb `io` verwijderd